test/cmd/diagnostics.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/cmd_util.sh"
+os::log::install_errexit
+
+# This test validates the diagnostics command
+
+# available diagnostics (2015-12-21):
+# AnalyzeLogs ClusterRegistry ClusterRoleBindings ClusterRoles ClusterRouter ConfigContexts DiagnosticPod MasterConfigCheck MasterNode NodeConfigCheck NodeDefinitions UnitStatus
+# Without things feeding into systemd, AnalyzeLogs and UnitStatus are irrelevant.
+# ConfigContexts runs into some invalid contexts created by other tests, so is omitted.
```

Comment:
`oc config` doesn't remove anything and I don't wanna parse this in bash or shell out to a better language. I can literally just `sed` out the 8 lines of offending contexts, though that seems fragile.